### ISSUE TITLE
Typescript error with `Symbol.matchAll`

### ISSUE THREAD

**Issue by IlCallo** [vuejs/composition-api#469]
Hello there, since beta-7 this error appears when building a project using this package:
```ts
ERROR in /node_modules/@vue/composition-api/dist/index.d.ts(179,13):
TS2339: Property 'matchAll' does not exist on type 'SymbolConstructor'
```

It appears when `"target": "es6"` or lower is used, instead of `"target": "esnext"`

**Comment by DarkLite1** [comment]
@pikax can it be that this is not fully fixed in `"@vue/composition-api": "1.0.0-beta.8"` ? I still get the error:

>ERROR in T:/Test/Brecht/Node/prod/hip-frontend/node_modules/@vue/composition-api/dist/index.d.ts(177,13):
TS2339: Property 'matchAll' does not exist on type 'SymbolConstructor'.

**Comment by pikax** [comment]
@DarkLite1 it wasn't published.

If not fixed on `1.0.0-beta.9` please let us know.

**Comment by IlCallo** [comment]
They haven't released the new beta version :)
